test/nginx/test-nginx.js

```diff
@@ -437,6 +437,7 @@ describe('nginx config', () => {
       [ '/-/x/images/icon_180x180.png',                    'revalidate' ],
       [ '/-/x/images/favicon.ico',                         'revalidate' ],
       [ '/-/x/locales/build/en/translation-combined.json', 'revalidate' ],
+      [ '/-/x/0n1W082ZWvx1O7XDsmHNqfwSrIjeeIH',            'revalidate' ],
```

Comment:
```suggestion

```

Reply:
Can this be moved to L427, so it's clear what it's replacing?

And then maybe add comment:

```suggestion
      [ '/-/x/0n1W082ZWvx1O7XDsmHNqfwSrIjeeIH',            'revalidate' ], // offline form
```